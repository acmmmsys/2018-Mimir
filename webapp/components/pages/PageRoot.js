import React, { Component } from 'react'
import 'style/pages/PageRoot.scss';

const PageRoot = props =>
    <article>
        { props.children }
    </article>

export default PageRoot;