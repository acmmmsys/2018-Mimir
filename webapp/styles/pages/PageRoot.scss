@import '../colors';
@import "~compass-mixins/lib/compass";

article {
    position: absolute;
    top: 0;
    bottom: 0;
    right: 0;
    left: 18%;
    overflow: auto;
}