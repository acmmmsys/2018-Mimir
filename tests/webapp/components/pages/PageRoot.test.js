import React from 'react';
import PageRoot from '../../../../webapp/components/pages/PageRoot'
import renderer from 'react-test-renderer';

let wrapper;

test('PageRoot does render', () => {
    wrapper = shallow(
        <PageRoot>

        </PageRoot>
    );
    expect(wrapper).toMatchSnapshot();
});
