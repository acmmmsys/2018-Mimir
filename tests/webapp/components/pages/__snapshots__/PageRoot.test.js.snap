// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageRoot does render 1`] = `<article />`;
